docs/data/date-pickers/accessibility/accessibility.md

```diff
@@ -0,0 +1,96 @@
+|                                <kbd class="key">Tab</kbd> | Moves focus to the next element        |
+|   <kbd class="key">Shift</kbd>+<kbd class="key">Tab</kbd> | Moves focus to the previous element    |
+| <kbd class="key">Arrow Keys (Left, Right, Up, Down)</kbd> | Moves focus among the available values |
+
+In addition to the basic keyboard support above, the day view component implements its own keyboard interactions:
+
+|                             Keys | Description                                                                       |
+| -------------------------------: | :-------------------------------------------------------------------------------- |
+|   <kbd class="key">Page Up</kbd> | Moves calendar to next month, keeping focus on the same day                       |
+| <kbd class="key">Page Down</kbd> | Moves calendar to previous month, keeping focus on the same day                   |
+|      <kbd class="key">Home</kbd> | Moves focus to first day of the week, tresspassing to previous month if necessary |
+|       <kbd class="key">End</kbd> | Moves focus to last day of the week, tresspassing to next month if necessary      |
+
+### Date Picker
+
+The [Date Picker](/x/react-date-pickers/date-picker/) combines the functionalities of the Date Field and Date Calendar components.
+
+Depending on which component is in focus, the Picker will provide the corresponding keyboard support, either from [DateField](/x/react-date-pickers/accessibility/#fields) or [DateCalendar](/x/react-date-pickers/accessibility/#date-calendar) and any changes made to the value will be consistently reflected across both components.
+
+### Date Range Calendar
+
+The Date Range Calendar implements a similar keyboard support as the day view of the [DateCalendar](/x/react-date-pickers/accessibility/#date-calendar) component, with a difference on the navigation among the previous and next months that must be achieved using the auxiliar arrows.
+
+|                                                      Keys | Description                               |
+| --------------------------------------------------------: | :---------------------------------------- |
+| <kbd class="key">Arrow Keys (Left, Right, Up, Down)</kbd> | Moves focus among the available values    |
+|                            <kbd class="key">Page Up</kbd> | Moves focus to the last day of the month  |
+|                          <kbd class="key">Page Down</kbd> | Moves focus to the first day of the month |
+|                               <kbd class="key">Home</kbd> | Moves focus to first day of the week      |
+|                                <kbd class="key">End</kbd> | Moves focus to last day of the week       |
+
+### Date Range Picker
+
+The [Date Range Picker](/x/react-date-pickers/date-range-picker/) integrates the features of both the Date Range Field and Date Range Calendar components. However it doesn't inherently provides every keyboard support implemented in these components.
```

Comment:
WDYT about not talking about the calendar here, because it's keyboard interaction is never active in a range picker. 🤔 
```suggestion
The [Date Range Picker](/x/react-date-pickers/date-range-picker/) keeps the focus on the field all the time and thus has the same keyboard navigation as Date Range [Field](/x/react-date-pickers/accessibility/#fields).
```

Reply:
You suggestion is much clearer ! 🎉